vcs-server/Dockerfile

```diff
@@ -18,7 +19,9 @@ auth-access = write' /home/svn/dist/conf/svnserve.conf &&\
 
 RUN mkdir /home/git/ &&\
     git config --global user.email 'test@example.com' &&\
-    git init /home/git/empty.git && cd /home/git/empty.git && git commit --allow-empty --message "Initial commit" && git branch gh-pages && git branch asf-site &&\
+    git init /home/git/empty.git && cd /home/git/empty.git &&\
+    ([ "x$GIT_DEFAULT_BRANCH" = "xmaster" ] || git checkout -b "$GIT_DEFAULT_BRANCH")&&\
```

Comment:
I guess `git branch -m ` should be used to rename the default branch name (e.g. after initial commit)

Reply:
Why is that? I believe both will have the same end result. After `git init` is called, there's no commits so `master` doesn't actually exist yet. I can change if you prefer though.

Reply:
Apparently another option in git 2.28+ is `git config --global init.defaultBranch $GIT_DEFAULT_BRANCH`